### PR TITLE
dinamic rules: multiple path

### PR DESCRIPTION
**Pull request by Hidanio**
More about feature [here](https://github.com/VKCOM/noverify/issues/1208).

### CODE REVIEW

File: src/linter/utils.go [hunk 1]
```diff
@@ -494,10 +494,28 @@ func cloneRulesForFile(filename string, ruleSet *rules.ScopedSet) *rules.ScopedS
...
-			if !strings.Contains(filename, rule.Path) || isFilePathExcluded(filename, rule) {
+			if isFilePathExcluded(filename, rule) {
 				continue
 			}
-			res = append(res, rule)
+
+			match := false
+
+			if rule.Paths == nil {
+				if strings.Contains(filename, "") {
```
Comment (Danil42Russia) [inline src/linter/utils.go:504]: doesn't make sense, will always be true

Comment (Hidanio) [reply src/linter/utils.go]: Thank you. Removed

File: src/linter/utils.go [hunk 2]
```diff
@@ -494,10 +494,20 @@ func cloneRulesForFile(filename string, ruleSet *rules.ScopedSet) *rules.ScopedS
...
 		res := make([]rules.Rule, 0, len(ruleByKind))
 		for _, rule := range ruleByKind {
-			if !strings.Contains(filename, rule.Path) || isFilePathExcluded(filename, rule) {
+			if isFilePathExcluded(filename, rule) {
 				continue
 			}
-			res = append(res, rule)
+
+			if rule.Paths == nil {
+				res = append(res, rule)
```
Comment (Danil42Russia) [inline src/linter/utils.go:502]: append `continue`

Comment (Hidanio) [reply src/linter/utils.go:502]: Fixed

**Review by Danil42Russia — APPROVED** [review]

**Pull request merged**